src/main/webapp/user-profile-page.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="user_script.js"></script>
+    <script src="dashboard-script.js"></script>
+    <script src="itempagescript.js"></script>
+    <title>Entertainment Hub - Profile</title>
+  </head>
+  <body onload="loadUserProfilePage()">
+    <div id="navbar"></div>
+    <div class="container">
+      <h1 class="text-center">User Profile Page</h1>
+      <div class="card mb-3" style="max-width: 1000px; margin: 0 auto;">
+        <div class="row no-gutters">
+          <div class="col-md-4">
+            <img id="avatar" src="" class="card-img-top" alt="profile avatar" />
+          </div>
+          <div class="col-md-8">
+            <div class="card-body">
+              <h6 class="card-title mb-2" id="username"></h6>
+              <p class="card-text" id="bio"></p>
+            </div>
+          </div>
+        </div>
+        <div class="mt-3">
+          <h2 class="text-center">Favorite Items</h2>
+          <div style="width: 45em; margin: 0 auto;" style="height: 10rem;">
+            <div id="item-container"></div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
```

Comment:
can we place these script tags with the others inside <head>?

Reply:
I moved them right inside the head tag in last commit.